DPG/Tasks/AOTTrack/qaImpPar.cxx

```diff
@@ -154,11 +158,12 @@ struct QaImpactPar {
   void processData(o2::soa::Filtered<collisionRecoTable>::iterator& collision,
                    const trackTable& tracksUnfiltered,
                    const o2::soa::Filtered<trackFullTable>& tracks,
+                   const o2::aod::TracksIU& tracksIU,
```

Comment:
in this way the `TracksIU` table is not filtered, is it what you want?

Reply:
Ciao @mfaggin, indeed this is one of my doubts I would prefer to filter the table in order to reduce its size to speed up the execution. Is it equivalent applying the track filter defined in the code on the track table with track parameters at collision vertex or on the track table with track parameters at IU? Just to be sure, because I was wandering that some parameters could have different values among the two tables

Reply:
Ciao @scattaru , indeed this is a good point especially because you extract `x`, `y`, `z` for the `trackIU` associated to each selected global track, and filtering the `tableIU` you might miss the corresponding `trackIU` for your global track (and instead if you analyse a `trackIU` not associated to any global track the `x`, `y`, `z` remain `-999.f`, so you recognize them offline).
Fair enough, let's go ahead in this way. I approve it.